k8s/resource.py

```diff
@@ -297,7 +302,7 @@ def _wait_statefulset_complete(self, kube_client, tries, timeout):
             if current_revision == update_revision:
                 log.info('desiredReplicas = {}, updatedReplicas = {}, availableReplicas = {}'.
                          format(replicas[0], replicas[1], replicas[2]))
-                if self._replicas_are_equal(replicas):
+                if self._replicas_are_greater_or_equal(replicas):
```

Comment:
Are HPA applicable to statefulsets or daemonsets?

Reply:
even if not,
it seems to me to do a separate function is not necessary